src/bots/depositor_bot.py

```diff
@@ -269,36 +276,47 @@ def _high_gas_fee_issue(self) -> bool:
             })
             return True
 
-    def _prohibit_to_deposit_issue(self) -> bool:
-        try:
-            can_deposit = contracts.deposit_security_module.functions.canDeposit(MODULE_ID).call(block_identifier=self.current_block.hash.hex())
-            logger.info({'msg': 'Call canDeposit().', 'value': can_deposit})
-
-            if not can_deposit:
-                logger.warning({'msg': self.DEPOSIT_SECURITY_ISSUE, 'value': can_deposit})
-                return True
+    def _prohibit_to_deposit_issue(self, stacking_module_id: int) -> bool:
+        can_deposit = contracts.deposit_security_module.functions.canDeposit(stacking_module_id).call(
+            block_identifier=self.current_block.hash.hex(),
+        )
+        logger.info({'msg': 'Call canDeposit().', 'value': can_deposit})
 
-        # TODO: get rid of try catch after the V2 upgrade
-        except Exception as error:
-            logger.info({'msg': 'canDeposit call exception.', 'error': str(error)})
-            return False
+        if not can_deposit:
+            logger.warning({'msg': self.DEPOSIT_SECURITY_ISSUE, 'value': can_deposit})
+            return True
 
-    def _available_keys_issue(self) -> bool:
-        available_keys = contracts.node_operator_registry.functions.assignNextSigningKeys(1).call(
-            {'from': contracts.lido.address},
+    def _available_keys_issue(self, stacking_module_id: int) -> bool:
+        depositable_ether = contracts.lido.functions.getDepositableEther().call(
```

Comment:
We also need to use `getDepositableEther` instead of `getBufferedEther` in `_buffered_ether_issue` method

Reply:
for now I think it's ok to leave the old strategy but with `getDepositableEther`, but later we need to rethink it, because for some modules, like Community, deposits may be small

Reply:
Fixed